commands/trigger_test.go

```diff
@@ -0,0 +1,67 @@
+package commands_test
```

Comment:
I think this would compile if you change the package to `commands` instead and remove `. "../commands"` from the imports.